Kinvey/Kinvey.csproj

```diff
@@ -11,7 +11,7 @@
     <PackageReference Include="SQLite.Net-PCL" Version="3.1.1" />
     <PackageReference Include="SQLite.Net.Async-PCL" Version="3.1.1" />
     <PackageReference Include="Remotion.Linq" Version="2.2.0" />
-    <PackageReference Include="Pubnub" Version="4.0.21" />
+    <PackageReference Include="Pubnub" Version="4.0.22" />

```

Comment:
Why does the `Pubnub` version have to be bumped up here?

Reply:
I got a crash because of PubNub...

https://cirrus-ci.com/task/4644658730762240